.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "stable"
+services:
+  - postgresql
+notifications:
+  email: false
+  
```

Comment:
A new line should be added at the end of every file

Reply:
I added a new line. @daylay92

Reply:
okay

Reply:
Since the package.json file has been updated alongside the test script , I think you can now update the .travis.yml file with the install and script properties like so:
**install : **
**npm install**
**script : **
**npm test**